unattended_installer/install_functions/indexer.sh

```diff
@@ -154,6 +154,19 @@ function indexer_install() {
 
 function indexer_startCluster() {
 
+    retries=0    
+    for ip_to_test in "${indexer_node_ips[@]}"; do
+        until eval "nc -z ${ip_to_test} 9300" || [ "${retries}" -eq 12 ]; do
+            sleep 10
+            retries=$((retries+1))
+        done
+        if [ ${retries} -eq 12 ]; then
+            common_logger -e "Cannot initialize Wazuh indexer cluster. Unable to connect to node ${ip_to_test} on port 9300."
+            installCommon_rollBack
```

Comment:
Please avoid the rollBack in distributed mode.

Reply:
Done: https://github.com/wazuh/wazuh-packages/commit/e5962cd7f8400b1e89f4210190215cb3b6c09c05